sdk/core/Azure.Core/perf/DownloadHttpClientTest.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.Pipeline;
+using Azure.Identity;
+using Azure.Test.Perf;
+
+namespace Azure.Template.Perf
+{
+    public class DownloadHttpClientTest : PerfTest<DownloadTestOptions>
+    {
+        private readonly HttpClient _client;
+        private readonly InProcTestServer _server;
+
+        public DownloadHttpClientTest(DownloadTestOptions options) : base(options)
+        {
+            _server = InProcTestServer.CreateStaticResponse(options.Size);
+            _client = new HttpClient();
+        }
+
+        public override void Run(CancellationToken cancellationToken)
+        {
+#if NET5_0
+            using var response = _client.Send(new HttpRequestMessage(HttpMethod.Get, _server.Address), Options.Buffer ? HttpCompletionOption.ResponseContentRead : HttpCompletionOption.ResponseHeadersRead, cancellationToken);
```

Comment:
Out of curiosity, why are we testing just plain ol' `HttpClient` instead of our pipeline that wraps it? Just to get some baselines?

Reply:
We are testing both. But we have HttpClient version for the baseline.